lib/active_model_serializers.rb

```diff
@@ -14,7 +14,7 @@
       ActionController::Base.send(:include, ::ActionController::Serialization)
       ActionController::TestCase.send(:include, ::ActionController::SerializationAssertions)
     end
-    ActionDispatch::Reloader.to_prepare do
+    ActiveSupport::Reloader.to_prepare do
```

Comment:
Can we move this to a Railtie and use the `to_prepare` in the application object to do this? This way it can support any Rails version, since ActiveSupport::Reloader doesn't exist on Rails 4.2 and 0.9 supports 4.2.